src/libraries/System.Private.Xml/src/System/Xml/XPath/XPathNavigator.cs

```diff
@@ -470,7 +471,7 @@ public virtual string LookupNamespace(string prefix)
             return null;
         }
 
-        public virtual string LookupPrefix(string namespaceURI)
+        public virtual string? LookupPrefix(string namespaceURI)
```

Comment:
should namespaceURI be nullable? here and in all methods here

Reply:
I agree it should be `string?` given the null check in the next line.
Should we also change `IXmlNamespaceResolver`?
https://github.com/dotnet/runtime/blob/4ac596dba139df2789965c094bb2b58607fa3a9e/src/libraries/System.Private.Xml/src/System/Xml/IXmlNamespaceResolver.cs#L34

Reply:
For the rest of the methods taking `string namespaceURI` as a parameter, I don't see any null-check nor a null value being received so I would suggest to keep them as not-null.

Reply:
Actually looking at the method name and disregarding implementation it doesn't make sense for this to accept null so I'm fine with this being as is (plus we can always make arguments nullable in the future and not reverse)